lib/farscape/transition.rb

```diff
@@ -16,11 +16,11 @@ def invoke(args={})
       options = match_params(args, opts)
 
       call_options = {}
-      call_options[:url] = @transition.uri
-      call_options[:method] = @transition.interface_method
+      call_options[:url] = @transition.uri || ''
+      call_options[:method] = @transition.interface_method || 'get'
```

Comment:
Yes, but, it seemed to create consistency with the other lines, and strikes me as more robust.